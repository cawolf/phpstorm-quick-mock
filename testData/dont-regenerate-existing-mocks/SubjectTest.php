<?php

namespace App\Tests;

use App\SubjectUnderTest;
use Prophecy\Prophecy\ObjectProphecy;
use Psr\Log\LoggerInterface;

class SubjectTest
{
    /** @var LoggerInterface|ObjectProphecy */
    private $logger;
    /** @var int */
    private $i;
    /** @var SubjectUnderTest */
    private $subject;

    public function setUp()
    {
        parent::setUp();
        $this->logger = $this->prophesize(LoggerInterface::class);
        $this->i = 0;

        $this->subject = new SubjectUnderTest<caret>($this->logger->reveal(), $this->i);
    }
}
