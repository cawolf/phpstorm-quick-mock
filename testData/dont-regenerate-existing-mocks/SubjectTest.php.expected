<?php

namespace App\Tests;

use App\SubjectUnderTest;
use Prophecy\Prophecy\ObjectProphecy;
use Psr\Log\LoggerInterface;

class SubjectTest
{
    /** @var string */
    private $s;
    /** @var LoggerInterface|ObjectProphecy */
    private $logger;
    /** @var int */
    private $i;
    /** @var SubjectUnderTest */
    private $subject;

    public function setUp()
    {
        parent::setUp();
        $this->logger = $this->prophesize(LoggerInterface::class);
        $this->i = 0;
        $this->s = '';

        $this->subject = new SubjectUnderTest($this->logger->reveal(), $this->i, $this->s);
    }
}